RecoEgamma/EgammaElectronProducers/python/gsfElectronSequence_cff.py

```diff
@@ -7,31 +7,14 @@
 #==============================================================================
 
 from RecoEgamma.EgammaElectronProducers.ecalDrivenGsfElectronCores_cfi import ecalDrivenGsfElectronCores
-from RecoEgamma.EgammaElectronProducers.gsfElectronCores_cfi import gsfElectronCores
 from RecoEgamma.EgammaElectronProducers.ecalDrivenGsfElectronCoresFromMultiCl_cff import ecalDrivenGsfElectronCoresFromMultiCl
 from RecoEgamma.EgammaElectronProducers.gsfElectrons_cfi import *
-gsfElectronTask = cms.Task(ecalDrivenGsfElectronCores,ecalDrivenGsfElectrons,gsfElectronCores,gsfElectrons)
-gsfElectronSequence = cms.Sequence(gsfElectronTask)
 
 gsfEcalDrivenElectronTask = cms.Task(ecalDrivenGsfElectronCores,ecalDrivenGsfElectrons)
 gsfEcalDrivenElectronSequence = cms.Sequence(gsfEcalDrivenElectronTask)
```

Comment:
This sequence seems to be only used in a few (outdated?) test scripts.
To be possibly considered in a future further cleaning campaing, while I agree that it could add unnecessary complications to add it here.